src/core/lib/event_engine/uv/libuv_event_engine.cc

```diff
@@ -303,7 +305,9 @@ EventEngine::TaskHandle LibuvEventEngine::RunAt(absl::Time when,
   if (now >= when) {
     timeout = 0;
   } else {
-    timeout = (when - now) / absl::Milliseconds(1);
```

Comment:
Why not: `absl::ToInt64Milliseconds(when - now) + 1`

Reply:
Done.

Reply:
(why + 1?)

And this was the first of the two documented method by Abseil:

![image](https://user-images.githubusercontent.com/7281574/135362706-c8bd1e7e-b8ef-4f29-85dd-a8f06bb5c810.png)

Reply:
See Craig's comment below for justification. The alternative I had previously solved the same problem without the arbitrary +1: `std::ceil(absl::ToUnixMicros(when) / 1000.0) - absl::ToUnixMillis(now);`. It's a more accurate calculation, but more expensive to calculate, and the +1ms is fairly trivial.

Reply:
Oh, to give a concrete example: in some cases, timers were firing 1ms before `absl::Now()` came around, due to the decimal truncation and millisecond resolution of the libuv EE, and the nanosecond resolution of absl's Time.